### PR TITLE
feat: add introduction to data model chapter

### PR DESCRIPTION
**Pull request by raimundo-henriques**
Updated version without diagram

### CODE REVIEW

**Comment by raimundo-henriques** [comment]
Thank you @GabrielaRubioDomingo ! Updated accordingly

**Comment by raimundo-henriques** [comment]
You're right, @GabrielaRubioDomingo! Thank you!
I've removed it now

**Review by GabrielaRubioDomingo — APPROVED** [review]

**Comment by raimundo-henriques** [comment]
Thanks for the approval, @GabrielaRubioDomingo . I've added a changelog entry and removed the 'Consultation Draft' tag. @zeitgeist , can you please double-check if the version number is correct?
Thanks!

File: specs/index.bs [hunk 1]
```diff
@@ -562,6 +562,21 @@ Additionally, the concept of transport activity data ([[#dt-tad|TAD]]) is added
...
+The iLEAP Data Model is composed by five main data types: <{ShipmentFootprint}>, <{TCE}>, <{TOC}>,
+<{HOC}>, and <{TAD}>.
+
+<{ShipmentFootprint|ShipmentFootprints}> and <{TOC|TOCs}> can be exchanged independently as extensions to the Pathfinder
+Data Model ([[!PACTDX]]) (see [[!DATA-MODEL-EXTENSIONS]]) in a `ProductFootprint`.
+
+<{TAD|TADs}> can be exchanged through a dedicated endpoint. These should only be used by [=Transport
+Operators=] who cannot yet provide <{ShipmentFootprint|ShipmentFootprints}>, <{TOC|TOCs}>, nor <{HOC|HOCs}>.
+
+The relations between these data types are summarized by the diagram below:
```
Comment (zeitgeist) [inline specs/index.bs:574]: I think this sentence is stray / no longer needed. WDYT @raimundo-henriques ?

File: specs/index.bs [hunk 2]
```diff
@@ -562,6 +562,21 @@ Additionally, the concept of transport activity data ([[#dt-tad|TAD]]) is added
...
+
+Note: A <{TCE}> MUST use either a <{TOC}> or an <{HOC}> (see [[#dt-tce]]). The dashed line aims at
```
Comment (zeitgeist) [inline specs/index.bs:576]: This paragraph also appears as redundant to me.

**Review by zeitgeist — CHANGES_REQUESTED** [review]
Looks good to me modulo the comments.

File: specs/index.bs (hunk 1 above)
Comment (raimundo-henriques) [reply specs/index.bs:574]: Absolutely. Thank you!

File: specs/index.bs (hunk 2 above)
Comment (raimundo-henriques) [reply specs/index.bs:576]: Absolutely! Thanks!

**Review by zeitgeist — APPROVED** [review]

**Pull request merged**